VideoLocker/src/main/java/org/edx/mobile/module/db/IDatabase.java

```diff
@@ -359,5 +359,10 @@ DownloadedState getDownloadedStateForVideoId(String videoId,
      */
     public Boolean isAnyVideoDownloadingInSubSection(DataCallback<Boolean> callback, String courseId,
                                                      String section, String subSection);
-    
+
+    /**
+     *
+     * @param userName
+     */
+    public void setUserName(String userName);
```

Comment:
The username won't actually change after we create this, right? Or is there currently an issue with login/logout?